### PR TITLE
Update node version from 12.18.0 to 12.18.4 across the whole repo

### PR DESCRIPTION
**Pull request by scinos**
### Background

A high severity issue (CVE-2020-8201) was found in all Node version, Node released patched versions for v10,x, v12.x and v14.x lines

https://nodejs.org/en/blog/vulnerability/september-2020-security-releases/

### Changes

Update Node from 12.18.0 to 12.18.4 across the whole repo.

### Testing instructions

Verify all tests pass

### CODE REVIEW

**Comment by scinos** [comment]
Blocked by #45722

**Review by tyxla — APPROVED** [review]
Looks good to me 👍  Thank you!

Do we need to also update the `engines` entry in `package.json`?

**Comment by scinos** [comment]
> Do we need to also update the engines entry in package.json?

Good catch!. We don't _have to_, but I guess it is more consistent.

File: .teamcity/settings.kts [hunk 1]
```diff
@@ -176,7 +175,8 @@ object RunAllUnitTests : BuildType({
 				export NODE_ENV="test"
 
 				# Update node
-				. "${'$'}NVM_DIR/nvm.sh"
+				. "${'$'}NVM_DIR/nvm.sh" --no-use
```
Comment (scinos) [inline .teamcity/settings.kts:178]: I'm not 100% sure why, but the previous code was not picking the new node version. I tested this config using TeamCity custom builds + a patch, and it seems to work.

**Pull request merged**